lib/voice_manager.py

```diff
@@ -0,0 +1,102 @@
+"""
+voice関連の処理を掌握するモジュール
+"""
+import asyncio
+
+import discord
+
+class VoiceManager():
+    """
+    音声関連の処理をするクラス
+    """
+
+    def __init__(self):
+        """
+        初期化処理
+        """
+        self.member = None
+        self.before = None
+        self.after = None
+        self.is_join = None
+
+    async def diff_embed(self, member: discord.Member, before: discord.VoiceState, after: discord.VoiceState):
+        """
+        embedを送信する関数
+        Parameters
+        ----------
+        member: discord.Member
+            該当するメンバーののオブジェクト
+        before: discord.VoiceState
+            変更前のVoiceStateオブジェクト
+        after: discord.VoiceState
+            変更後のVoiceStateオブジェクト
+        """
+        self.member = member
+        self.before = before
+        self.after = after
+
+        if after.channel == before.channel or member.id in [684295118643265548]:
+            return
+        if after.channel is None:
+            self.is_join = False
+            await VoiceManager.KIKISEN_CHANNEL.send(embed=self.embed_generater())
+            await self.role_manager()
+            return
+        self.is_join = True
+        await VoiceManager.KIKISEN_CHANNEL.send(embed=self.embed_generater())
+        await self.role_manager()
+
+    def embed_generater(self):
+        """
+        instanceの状況に応じてembedを返す関数
+        """
+        if self.is_join:
+            embed_in = discord.Embed(title="{}が{}に入りました".format(self.member.display_name, str(self.after.channel)),
+                                     description="何かが始まる予感がする。", color=0x1e63e9)
+            embed_in.set_author(name="はらちょからのおしらせ", icon_url="https://lohas.nicoseiga.jp/thumb/3877931i?")
+            embed_in.set_thumbnail(
+                url="https://cdn.discordapp.com/avatars/{}/{}.png".format(self.member.id, self.member.avatar))
+            return embed_in
+
+        embed_out = discord.Embed(title="{}が{}から抜けました".format(self.member.display_name, str(self.before.channel)),
+                                  description="あいつは良い奴だったよ...", color=0x1e63e9)
```

Comment:
```suggestion
        if self.is_join:
            embed_title = "{}が{}に入りました"
            embed_message = "何かが始まる予感がする。"
        else: #<--- !!!!!!!!!!!!!!!!!!!!!!!!!
            embed_title = "{}が{}にから抜けました"
            embed_message = "あいつは良い奴だったよ…"
        embed_out = discord.Embed(title=embed_title.format(self.member.display_name, str(self.before.channel)), description=embed_message, color=0x1e63e9)
```
ここ変数使ってどうにか出来ないかなって思ってコメントつけようとしたんですけど`else警察`の存在を思い出したのでなんでもないです
三項演算子でかいくぐろうかと思ったけどウンコードだしPythonの三項演算子カスだし警察いそうなのでいいです